codemods/callouts.js

```diff
@@ -0,0 +1,41 @@
+const visit = require('unist-util-visit');
+const {
+  addAttribute,
+  isMdxBlockElement,
+  removeAttribute,
+} = require('./utils/mdxast');
+
+const SPECIAL_VARIANTS = {
+  permissions: 'tip',
```

Comment:
We also want to use `tip` for `note` callouts. And since the Callout component doesn't support permissions or pricing callout variants (TBD by tech docs on how we handle those), I think it's good to just use `tip` variant for now.
```  permissions: 'tip',
note: 'tip',
pricing: 'tip',
```

Reply:
Perfect! I knew there were others, but couldn't find them because unfortunately the subset of docs we have don't have the other two. I'll update this to include the others. Thanks a bunch!

Reply:
If we do find that `permissions` is its own variant, we can absolutely update this codemod later.